docker-compose.yml

```diff
@@ -25,6 +26,14 @@ services:
       interval: 2s
       timeout: 3s
       retries: 15
+    volumes:
+      - postgres_data:/var/lib/postgresql/data
+
+  adminer:
+    image: adminer
+    restart: always
+    ports:
+      - 8080:8080
```

Comment:
Since this isn't necessary to run it, I think we should leave it out of the basic docker setup